port/unix/omrsysinfo.c

```diff
@@ -253,6 +253,18 @@ static intptr_t omrsysinfo_get_aix_ppc_description(struct OMRPortLibrary *portLi
 #define __power_10() (_system_configuration.implementation == POWER_10)
 #endif /* !defined(__power_10) */
 
+#if !defined(__power_11)
+#define POWER_11 0x80000 /* Power 11 class CPU */
+#define __power_11() (_system_configuration.implementation == POWER_11)
+#endif /* !defined(__power_11) */
+
+/*
+ * Please update the macro below to stay in sync with the latest POWER processor known to OMR,
+ * ensuring CPU recognition is more robust than in the past. As the macro currently stands, any
+ * later processors are recognized as at least POWER11.
+ */
+#define __power_latestKnownAndUp() ((_system_configuration.implementation & (0xFFFFFFFF << 19)) != 0)
```

Comment:
Using a macro for `0xFFFFFFFF << 19` will give it meaning. The below suggestion will make the code more clear and reuse an existing macro (`OMR_ARE_ANY_BITS_SET`).
```suggestion
#define POWER_11_OR_ABOVE (0xFFFFFFFF << 19)
#define __power_latestKnownAndUp() OMR_ARE_ANY_BITS_SET(_system_configuration.implementation, POWER_11_OR_ABOVE)
```

Reply:
ok ... will change to that